pkg/handlers/v1/subnet.go

```diff
@@ -0,0 +1,77 @@
+package v1
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+type subnetConfiguration struct {
+	CIDRBlock string `json:"cidrBlock"`
+}
+
+type subnetConfigurationDiff struct {
+	PreviousValue *subnetConfiguration `json:"previousValue"`
+	UpdatedValue  *subnetConfiguration `json:"updatedValue"`
+	ChangeType    string               `json:"changeType"`
+}
+
+type subnetTransformer struct{}
+
+func (t subnetTransformer) Create(event awsConfigEvent) (Output, bool, error) {
+	output, err := getBaseOutput(event.ConfigurationItem)
+	if err != nil {
+		return Output{}, false, err
+	}
+
+	var config subnetConfiguration
+	if err := json.Unmarshal(event.ConfigurationItem.Configuration, &config); err != nil {
+		return Output{}, false, err
+	}
+
+	change := extractSubnetInfo(&config)
+	change.ChangeType = added
+	output.Changes = append(output.Changes, change)
+	return output, false, nil
+}
+
+func (t subnetTransformer) Update(event awsConfigEvent) (Output, bool, error) {
```

Comment:
Instead of not doing anything with the update event should we consider capturing the changes coming with Update events? Might be helpful if we decide to use [other subnet properties](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-subnet.html)